src/tests/ftest/util/command_utils_base.py

```diff
@@ -789,6 +799,76 @@ def _get_new(self):
         return TransportCredentials(self.namespace, self.title, self._log_dir)
 
 
+class TelemetryCredentials(YamlParameters):
```

Comment:
name suggestion: TelemetryConfig

It looks like port, retain, enabled, etc. are a part of this. I would also avoid calling these certs "credentials" in general. They aren't used for access control or user identification.